it/v1/endpoints/DeleteReliefInvestmentsControllerISpec.scala

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIED OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.endpoints
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status
+import play.api.libs.json.{JsObject, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import support.IntegrationBaseSpec
+import v1.models.errors._
+import v1.stubs.{AuditStub, AuthStub, DesStub, MtdIdLookupStub}
+
+class DeleteReliefInvestmentsControllerISpec extends IntegrationBaseSpec {
+
+  private trait Test {
+
+    val nino = "AA123456A"
+    val taxYear = "2019-20"
+
+    def uri: String = s"/investment/$nino/$taxYear"
```

Comment:
reliefs/investment ?

Reply:
This would be the MTDAPI url right?
it omits individuals/reliefs from here